doc/manuals/admin/perf_tuning.md

```diff
@@ -40,19 +40,28 @@ speeds up reads) and write efficiency (the usage of indexes slows down writes) a
 consume space in database and mapped RAM memory) and it is the administrator (not Orion) who has
 to decide what to prioritize.
 
-However, in order to help administrators in this task, the following indexes are recommended:
+However, in order to help administrators in this task, the following *general* indexes are recommended:
 
 * Collection [entities](database_model.md#entities-collection)
-    * `_id.id`
-    * `_id.type`
-    * `_id.servicePath`
-    * `attrNames`
+    * `{_id.servicePath: 1, _id.id: 1, _id.type: 1}` (note this is a compound index and key order matters in this case)
     * `creDate`
 
-In the case of using `orderBy` queries (i.e. `GET /v2/entities?orderBy=A`), it is also recommended 
+In addition, depending on your queries you could need *additional* indexes.
```

Comment:
could => may
Also, I'd add a comma after queries:

In addition, depending on your queries, you may need *additional* indexes.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3495/commits/693187af336cc4a95a919d0b5e8eba2807ae05fd